From b51146493b5efb8bd9da900a132db39e4d729855 Mon Sep 17 00:00:00 2001
From: Omar Ramirez Luna <omar.ramirez@ti.com>
Date: Wed, 17 Mar 2010 18:48:10 -0600
Subject: [PATCH 003/135] DSPBRIDGE: Remove autoconf.h dependency from host_os header

There shouldn't be any dependency on the definitions generated
by autoconf.h, besides it is automatically included by the
build system.

This solves the following warning:

In file included from arch/arm/plat-omap/include/dspbridge/host_os.h:20,
                 from arch/arm/mach-omap2/io.c:51:
include/linux/autoconf.h:165:1: warning: "CONFIG_DEFAULT_IOSCHED" redefined
In file included from <command-line>:0:
./include/generated/autoconf.h:172:1: warning: this is the location of the previous definition

Signed-off-by: Omar Ramirez Luna <omar.ramirez@hotmail.com>
---
 arch/arm/plat-omap/include/dspbridge/host_os.h |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/arch/arm/plat-omap/include/dspbridge/host_os.h b/arch/arm/plat-omap/include/dspbridge/host_os.h
index 00ea285..92b12de 100644
--- a/arch/arm/plat-omap/include/dspbridge/host_os.h
+++ b/arch/arm/plat-omap/include/dspbridge/host_os.h
@@ -17,7 +17,6 @@
 #ifndef _HOST_OS_H_
 #define _HOST_OS_H_
 
-#include <linux/autoconf.h>
 #include <asm/system.h>
 #include <asm/atomic.h>
 #include <linux/semaphore.h>
-- 
1.7.0.4

